components/telemetry-operator/config/crd/bases/telemetry.kyma-project.io_logpipelines.yaml

```diff
@@ -0,0 +1,111 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: logpipelines.telemetry.kyma-project.io
+spec:
+  group: telemetry.kyma-project.io
+  names:
+    kind: LogPipeline
+    listKind: LogPipelineList
+    plural: logpipelines
+    singular: logpipeline
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: LogPipeline is the Schema for the logpipelines API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
```

Comment:
Absolutely not a fan of splitting sentences to different lines ("translators hate this trick", and even without translation, line breaks may look weird together on the automatic line breaks in output).
 
YAML doesn't enforce that, does it? If there's no technical reason to split the sentences, use one line per full sentence.

Reply:
The line wrapping is done by Kubebuilder (or more specifically `controller-gen`) automatically. I will check if there is a way to avoid this  👍

Reply:
Oh nooo. Thanks kubebuilder. Does it have reasons to enforce the line breaks? Don't bother if this is a lot of work, we're probably never going to translate this file.

Reply:
That is a bug. But I fixed it by upgrading `controller-gen` to `v0.5.0`.